### DISCUSSION TITLE
Problem of format in Timestamp

### DISCUSSION THREAD

**Discussion by Ange-Heureux** [pilotmoon/PopClip-Extensions#1155] [General]
Hello,
I'm an old user of PopClip but a new one of timestamp... and i have the following problem : 
the inserted date is Apr/Sun/2021 08:52:18 and
- no number, but letters ! (I've chosen M/D/Y)
- the current time is 10:52:18 !!!
- the most problem is the date !!!
For info, i'm on Big Sur on M1 mini mac.
Could explain me how to correct this problem ?
Thanks in advance.
Ange
PS : possible to add the FR format ? D/M/Y (with 0 : 25/04/2021)

**Comment by pilotmoon** [comment]
Thanks for the feedback! I have updated the date formats available (commit acd7405) - now d/m/y will give e.g. 26/04/2021

Next problem is the time zone. Unfortunately the time is always prineted as UTC time. This will be harder to fix! I will see wehat I can do....

**Reply by pilotmoon** [reply]
Time zone should also be fixed now. You'll need to redownload from https://pilotmoon.com/popclip/extensions/page/Timestamp

**Reply by Ange-Heureux** [reply]
Thanks for the reactivity ! But sorry I can't test ! Timestamp doesn't appear anymore in PopClip (without anything selected as before, nor if something is selected !). I've deleted explicitly the Timestamp module, reinstalled it from the proposed link, even restart the mac, but timestamp doesn't appear anymore :-( !

**Reply by pilotmoon** [reply]
Oh - it looks like I made a silly mistake. Please try again now.

**Comment by Ange-Heureux** [comment]
Hi
Thanks ! 
Great ! all runs.

So, i permit me to ask you a future enchantement : to have the possibility to chose direclty with or without the time ? (but perhaps it means to have 2 modules ? but in fact i think that i will only use the date (without time), so no problem for me ;-))

Test for info : 
26/04/2021 17:59

Thanks again,
Ange

**Comment by Ange-Heureux** [comment]
Hi Nick,

I’m currently testing the Beta. Great Job, I was suprised that my own extensions (search info into specific sites, done with PopMaker) were directly compatible with the new script/PHP system ! (and Timestamp too ! ;-)). great.

I’m waiting for next delivery for some extensions (as Calculate) but don’t worry, i don’t use it all days, i can wait !

Thank you again for PopClip, always very useful !

Regards,
Ange

**Reply by pilotmoon** [reply]
Thanks for letting me know. I am surprised that these PHP extensions are working on Monterey for you. Can you confirm this is the case - what exact version of macOS?

**Comment by Ange-Heureux** [comment]
Hi Nick,

and ToDoIst seems ok, although you write " Not yet compatible with macOS 12.0 Beta" !

Ange

**Comment by Ange-Heureux** [comment]
Hi Nick,

Sorry, I’m still on Big Sur (macOS 11.4 (20F71)) on a Mac Mini M1.

But as explained in an article about PopClip in the French VVMac site/pdf, we can test already the Beta & extensions….

Sorry for my misunderstanding !

Regards
Ange

**Reply by pilotmoon** [reply]
Ah, no problem at all. Yes, there is scope for confusion between PopCLip beta and macOS beta!
